polaris.shopify.com/src/components/IconDetails/IconDetails.module.scss

```diff
@@ -2,14 +2,15 @@
 @import "../../styles/mixins.scss";
 
 .Title {
+  @include heading-2;
   display: flex;
-  align-items: center;
-  @include heading-4;
-  gap: 0.5rem;
+  align-items: flex-start;
+  font-weight: var(--font-weight-400);
+  gap: 0.75rem;
 }
```

Comment:
The sidebar is already maxing out height and increasing these values is going to show much less on screens with a small height. This screenshot is the current example and the sidebar is already too large for my laptop. I can't press the propose change button.

<img width="1488" alt="Screen Shot 2022-06-29 at 9 41 30 PM" src="https://user-images.githubusercontent.com/19199063/176594323-051077ba-b061-4e48-9261-434f9cd097ce.png">